### DISCUSSION TITLE
Support for tabler.io icons

### DISCUSSION THREAD

**Discussion by dudelisius** [livewire/flux#273] [Feature Requests]
Although heroicons is a great set of icons it is very limited. In the past I have used https://tabler.io/icons in my projects and these work very well. Be able to select these (or other icon sets) would be awesome. 

Also a very big nice to have!

**Comment by mauritskorse** [comment]
I have created a laravel package for adding custom icons to flux: https://github.com/Ympact/flux-icons
It's the first version, so there might still be some bugs ;)
It currently only has the [Tabler](https://tabler.io/icons) icon-set, but other vendors can easily be added, although some tweaking might be necessary to make the icons properly work in Flux.
